src/index.html

```diff
@@ -1,92 +1,92 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>List of employees</title>
-    <link rel="stylesheet" href="./styles/main.scss">
-  </head>
-  <body>
-    <h1>List of employees</h1>
-    <ul>
-      <li
-        data-position="Accountant"
-        data-salary="$162,700"
-        data-age="33"
-      >
-        Airi Satou
-      </li>
-      <li
-        data-position="Javascript Developer"
-        data-salary="$183,000"
-        data-age="29"
-      >
-        Michael Bruce
-      </li>
-      <li
-        data-position="Marketing Designer"
-        data-salary="$198,000"
-        data-age="66"
-      >
-        Michael Silva
-      </li>
-      <li
-        data-position="Technical Author"
-        data-salary="$145,000"
-        data-age="27"
-      >
-        Prescott Bartlett
-      </li>
-      <li
-        data-position="Developer"
-        data-salary="$98,540"
-        data-age="61"
-      >
-        Thor Walton
-      </li>
-      <li
-        data-position="Integration Specialist"
-        data-salary="$327,900"
-        data-age="55"
-      >
-        Rhona Davidson
-      </li>
-      <li
-        data-position="Data Coordinator"
-        data-salary="$138,575"
-        data-age="64"
-      >
-        Serge Baldwin
-      </li>
-      <li
-        data-position="Regional Director"
-        data-salary="$183,000"
-        data-age="51"
-      >
-        Shad Decker
-      </li>
-      <li
-        data-position="Regional Marketing"
-        data-salary="$163,000"
-        data-age="20"
-      >
-        Shou Itou
-      </li>
-      <li
-        data-position="Financial Controller"
-        data-salary="$452,500"
-        data-age="62"
-      >
-        Vivian Harrell
-      </li>
-      <li
-        data-position="Javascript Developer"
-        data-salary="$205,500"
-        data-age="39"
-      >
-        Colleen Hurst
-      </li>
-    </ul>
-    <script type="text/javascript" src="scripts/main.js"></script>
-  </body>
+    <head>
+        <meta charset="UTF-8">
```

Comment:
Remove all changes from the HTML file, please. Seems that you have a default formatter. It's better to disable automatic formatting

Reply:
I made indentations in html how it was in initial file 

<img width="1549" alt="Screenshot 2023-06-18 at 5 34 11 PM" src="https://github.com/mate-academy/js_task_parse_list_DOM/assets/105941605/c9111ed3-5700-4cd9-be0c-02319928ed5a">